src/webapp/js/components/sidebar.js

```diff
@@ -241,20 +241,36 @@ class SideBar extends Component {
       <ul className="nav nav-second-level">
         { hasAuthority(MANAGE_USERS_AUTHORITY, MANAGE_INCHARGE_USERS_AUTHORITY) &&
         <li className="border-none">
-          <Link to="/users/new" onClick={this.props.hideMenuSmart}>
+          <Link to="/users/users/new" onClick={this.props.hideMenuSmart}>
```

Comment:
why do we need to change the url to 'users/users'?

Reply:
so 'users/roles' is the list of user's roles? could we have the list of users under '/users' (the parent)?

Reply:
no, the users/roles is the list of all roles, also there are users/user/new to create new user and users/users/{userId} to edit user and for roles users/role/new and users/role/{roleId}